app/assets/stylesheets/bali/code.scss

```diff
@@ -0,0 +1,99 @@
+.content {
```

Comment:
I think we should scope this whole file under a `code-syntax` class so that it doesnt get applied generally. 

Also .content is very generic and already used in bulma.